sufia.gemspec

```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |spec|
   spec.version       = version
   spec.license       = 'Apache2'
 
-  spec.add_dependency 'curation_concerns', '~> 0.4'
+  spec.add_dependency 'curation_concerns', '~> 0.5', '>= 0.5.0'
```

Comment:
I mistakenly thought that '~> 0.5' might allow 0.4, but no, that's not right. Removing that argument.